src/home/constants.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_COURSES_COUNT = 9;
```

Comment:
The default value was `9` because I noticed that during local development I was getting exactly `9` as the value for [HOMEPAGE_COURSE_MAX](https://github.com/openedx/edx-platform/blob/master/lms/envs/devstack.py#L212)

<img width="974" height="275" alt="Image" src="https://github.com/user-attachments/assets/75cb11a2-ec9b-401f-bfd1-df43e0f41cff" />